### ISSUE TITLE
Cheguei na aula de Issues agora, estou criando minha primeira :D

### ISSUE THREAD

**Issue by valdean132** [gustavoguanabara/git-github#574]
Suas aulas são tops professor Gustavo, sem dúvida nenhuma o melhor professor...

**Comment by Vjsantana** [comment]
Verdade! O cara é da hora. Muito melhor que certos professores da Faculdade.

**Comment by ricardosmotta** [comment]
Chegando agora nessa aula tbm... comentandoaqui  pq a issue #5 foi fechada kkkk
